tool/teleport/common/teleport.go

```diff
@@ -165,6 +172,17 @@ func Run(options Options) (executedCommand string, conf *service.Config) {
 	scpc.Flag("local-addr", "local address which accepted the request").StringVar(&scpFlags.LocalAddr)
 	scpc.Arg("target", "").StringsVar(&scpFlags.Target)
 
+	// dump flags
+	dump.Flag("cluster-name",
+		"Unique cluster name, e.g. example.com.").StringVar(&dumpFlags.ClusterName)
+	dump.Flag("output",
+		"Write to stdout, default config file or custom path").Short('o').Default(
+		teleport.SchemeStdout + "://").StringVar(&dumpFlags.output)
+	dump.Flag("acme",
+		"Get automatic certificate from Letsencrypt.org using ACME.").BoolVar(&dumpFlags.ACMEEnabled)
+	dump.Flag("acme-email",
+		"Email to receive updates from Letsencrypt.org.").StringVar(&dumpFlags.ACMEEmail)
+
```

Comment:
It looks like `SampleFlags` also have `AddLicense` field but there's no corresponding CLI flag. Do you need to add it as well?